src/pages/ConvertVrt/Convert/index.tsx

```diff
@@ -0,0 +1,204 @@
+}
+
+type FormatI18nextRelativetimeValuesReturn =
+  | {
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeHoursAndMinutes';
+      realtiveTimeFormatValues: { hours: number; minutes: number };
+    }
+  | {
+      relativeTimeTranslationKey:
+        | 'convertVrt.remainingTimeDays'
+        | 'convertVrt.remainingTimeHours'
+        | 'convertVrt.remainingTimeMinutes';
+      realtiveTimeFormatValues: { count: number };
+    }
+  | {
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeMissing';
+      realtiveTimeFormatValues: { count: string };
+    };
+// Don't remove this comment, it allows for translation keys set by this function to be collected
+// t('convertVrt.remainingTimeDays_one') t('convertVrt.remainingTimeDays_other') t('convertVrt.remainingTimeHours_one') t('convertVrt.remainingTimeHours_other')
+// t('convertVrt.remainingTimeHoursAndMinutes') t('convertVrt.remainingTimeMinutes') t('convertVrt.remainingTimeMinutes_other') t('convertVrt.minutes_one')
+// t('convertVrt.minutes_other') t('convertVrt.hours_one') t('convertVrt.hours_other') t('convertVrt.days_one') t('convertVrt.days_other')
+const formatI18nextRelativetimeValues = (
+  vrtConversionEndTime: string | undefined,
+): FormatI18nextRelativetimeValuesReturn => {
+  if (vrtConversionEndTime === undefined) {
+    return {
+      realtiveTimeFormatValues: { count: PLACEHOLDER_KEY },
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeMissing',
+    };
+  }
+  const MINUTE_IN_MILISECONDS = 60 * 1000;
+  const HOUR_IN_MILISECONDS = 60 * MINUTE_IN_MILISECONDS;
+  const DAY_IN_MILISECONDS = 24 * HOUR_IN_MILISECONDS;
+  const vestingTimeRemainingMs = new Date().getTime() - +vrtConversionEndTime * 1000;
+  let relativeTimeValues: FormatI18nextRelativetimeValuesReturn = {
+    realtiveTimeFormatValues: { count: Math.floor(vestingTimeRemainingMs / DAY_IN_MILISECONDS) },
+    relativeTimeTranslationKey: 'convertVrt.remainingTimeDays',
+  };
+  if (vestingTimeRemainingMs === HOUR_IN_MILISECONDS) {
+    relativeTimeValues = {
+      realtiveTimeFormatValues: { count: Math.floor(vestingTimeRemainingMs / HOUR_IN_MILISECONDS) },
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeHours',
+    };
+  } else if (vestingTimeRemainingMs < HOUR_IN_MILISECONDS) {
+    relativeTimeValues = {
+      realtiveTimeFormatValues: {
+        count: Math.floor(vestingTimeRemainingMs / MINUTE_IN_MILISECONDS),
+      },
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeMinutes',
+    };
+  } else if (vestingTimeRemainingMs < DAY_IN_MILISECONDS) {
+    relativeTimeValues = {
+      realtiveTimeFormatValues: {
+        hours: Math.floor(vestingTimeRemainingMs / HOUR_IN_MILISECONDS),
+        minutes: Math.floor(
+          ((vestingTimeRemainingMs % HOUR_IN_MILISECONDS) / HOUR_IN_MILISECONDS) * 60,
+        ),
+      },
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeHoursAndMinutes',
+    };
+  }
+  return relativeTimeValues;
```

Comment:
I saw you marked my previous comment on how to format the time as resolved, but this doesn't follow the rules I wrote:
- if time is longer than a day, display whole days only. .e.g: 2 days
- if time is longer than an hour but shorter than a day, display hours and minutes only. e.g.: 10 hours 30 minutes
- otherwise display whole minutes only. e.g.: 46 minutes

Reply:
It does follow the rules but the math is a bit off for example I assumed milliseconds but it is actually returned as a timestamp. This has been updated in the [PR](https://github.com/VenusProtocol/venus-protocol-interface/pull/422/files#diff-e7f075c5ecae4dbbe05afc73300eeafbcbb3e9ef4504e8c1f42528da1c25a3aeR57) where it wires everything up. 

We can also take those changes here.